README.md

```diff
@@ -166,6 +166,26 @@ It defaults to `setup-uv-cache` in the `TMP` dir, `D:\a\_temp\uv-tool-dir` on Wi
     cache-local-path: "/path/to/cache"
 ```
 
+### Disable cache pruning
+
+By default, the uv cache is pruned after every run, removing pre-built wheels, but retaining any
+wheels that were built from source. On GitHub-hosted runners, it's typically faster to omit those
+pre-built wheels from the cache (and instead re-download them from the registry on each run).
+However, on self-hosted or local runners, preserving the cache may be more efficient. See
+the[documentation](https://docs.astral.sh/uv/concepts/cache/#caching-in-continuous-integration) for
+more.
+
+If you want to persist the entire cache across runs, disable cache pruning with the `prune-cache`
+input.
+
+```yaml
+- name: Don't prune the cache before saving it
+  uses: astral-sh/setup-uv@v3
+  with:
+    enable-cache: true
+    prune-cache: false
+```
+
```

Comment:
WDYT about adding this example:

To only disable cache pruning when running locally with [`act`](https://github.com/nektos/act), you can check the `ACT` environment variable:

```yaml
- name: Only disable cache pruning on local runs
  uses: astral-sh/setup-uv@v3
  with:
    enable-cache: true
    prune-cache: ${{ env.ACT != 'true' }}
```

Reply:
I'd rather not start adding documentation on how this action can be used with other tools. I fear this can quickly get overwhelming when we also start adding e.g. more advices for act, for gitea,...

@charliermarsh could you please enable discussions in this repo so @merlinz01 can describe the problem and provide the solution above?